website/client/src/components/chat/chatCard.vue

```diff
@@ -278,6 +282,15 @@ export default {
       return date.toString();
     },
   },
+  beforeMount () {
+    const regexCheckLang = /^[\u0600-\u06FF]/;
```

Comment:
I'm wondering what's going to happen if someone posts a message with mostly latin characters but then adds just a word in Arabic for example.

Also I'm not familiar at all with RTL languages support but looking at links like https://stackoverflow.com/questions/12006095/javascript-how-to-check-if-character-is-rtl/14824756 I'm wondering if this regex is enough or if it'll only cover Arabic but not any other RTL language

Reply:
Honestly I don't know the range of other rtl languages so I can't test them like Arabic

Reply:
Also if a message has mixed languages it will be aligned to the right if it detects that the first character of the message is Arabic